adminsortable2/admin.py

```diff
@@ -115,7 +115,7 @@ def get_changelist(self, request, **kwargs):
     @property
     def media(self):
         m = super(SortableAdminMixin, self).media
-        if self.enable_sorting:
+        if getattr(self, "enable_sorting", False):
```

Comment:
Just a short note, I follow Douglas Crockford's convention to use double quotes for strings visible to the user and single quotes for internal strings. Even in Python.
https://plus.google.com/+DouglasCrockfordEsq/posts/EBky2K9erKt